Team Management Changes (#357)
gamemanual0/gm0: source/docs/being-a-team/outreach-basics.rst

Many teams often hope to create large-scale programs that impact tens of thousands. However, many smaller outreach programs can often be more impactful than one big, unwieldy program that may never get off the ground. Consider what your local area needs - maybe your library needs volunteers to teach STEM, or your local FLL team requires mentorship. Maybe there are no other FLL or FTC teams in your area. Regardless, your goal should be to advocate for STEM and FIRST in your area.

> .. important::
>    Create a plan! Any set of outreaches should be outlined in a business plan. While this plan should also comprise your team vision, budget goals, and growth objectives, you can also document outreach as a way to expand your program's reach.

Reviewer: ```suggestion
.. important:: Create a plan! Any set of outreaches should be outlined in a business plan. In addition to documenting how outreach serves as a way to expand your program's reach, this plan should contain your team vision, budget goals, and growth objectives.
```

      *The Pitch: Support local STEM students by allowing local robotics team to bag at your store!*

      - Visit your local supermarket and talk to a manager or customer service rep to politely introduce your team.
      - Respectfully ask if they would be willing to let you bag for customers and set up tip jars.

>       - Obtain contact information of the person you spoke to, and write a followup email to them thanking them and confirming a date and time.

Reviewer: ```suggestion
      - Obtain contact information of the person you spoke to, and write a follow-up email to them thanking them and confirming a date and time.
```
